modules/cloud-function/README.md

```diff
@@ -0,0 +1,160 @@
+# Cloud Function Module
+
+Cloud Function management, with support for IAM roles and optional bucket creation.
+
+The GCS object used for deployment uses a hash of the bundle zip contents in its name, which ensures change tracking and avoids recreating the function if the GCS object is deleted and needs recreating.
+
+## TODO
+
+- [ ] add support for source repository
```

Comment:
It would be nice to add support to vpc_connector. It can be done on a future iteration.

Reply:
Ah yes, I planned on adding support for ingress settings and vpc connector, then forgot. Let's merge this and I'll add in a second PR. Thanks for spotting this. :)